src/vm_ppc.dasc

```diff
@@ -977,16 +977,21 @@ static void build_subroutines(BuildCtx *ctx)
   |.if FFI
   |  cmplwi TMP0, 1
   |.endif
-  |     lwz PC, -16(RB)			// Restore PC from [cont|PC].
-  |   subi TMP2, RD, 8
+  |// PC value corrected to avoid segfault
```

Comment:
should this part be merged into git@github.com:LuaJIT/LuaJIT.git

Reply:
This change was already included in moonjit. 

https://github.com/moonjit/moonjit/blob/a2a39ea7184f3c8cab9474c6e41f6541265fb362/src/vm_ppc.dasc#L981